.github/workflows/pytest.yml

```diff
@@ -45,3 +45,12 @@ jobs:
           FIREWORKS_API_KEY: ${{ secrets.FIREWORKS_API_KEY }}
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
           TOGETHER_API_KEY: ${{ secrets.TOGETHER_API_KEY }}
+      - name: Run the documentation notebooks
+        run: poetry run pytest -v --nbmake ./docs/**/*ipynb
```

Comment:
Are you sure this is the right command? When I run it locally all the notebooks fail with errors like

`AttributeError: module 'pixeltable' has no attribute 'functions'`

I know that rag-demo, at least, should be working, and if not I wouldn't expect this to be the error?